tracing-subscriber/src/subscribe/mod.rs

```diff
@@ -1382,6 +1382,18 @@ pub trait Filter<S> {
         None
     }
 
+    /// Called before the filtered subscribers' [`on_event`], to determine if
+    /// `on_event` should be called.
+    ///
+    /// This gives a chance to filter events based on their fields.
+    ///
```

Comment:
should maybe mention that this just returns `true` by default?

Reply:
👍